### ISSUE TITLE
Grey's Anatomy not working

### ISSUE THREAD

**Issue by Amelandbor** [cflannagan/service.subtitles.addic7ed#4]
First of all thanks for the hard work to code this great addon.

Grey's Anatomy is not working, probably something to do with the '
I see I could add it to addic7ize, but the name seems the same on addic7ed. 

Also wouldn't it be easier to pull the addic7ize dictionary from a different textfile, that way it's easier to maintain without editing inside a code file. Or maybe it's even easier to use a mapping by addic7ed id. Something like http://code.google.com/p/autosub-bootstrapbill/ is using.

Also a small thing I noticed from your readme, the folder should be cloned into to addon folder instead of the addon_data folder.

**Comment by cflannagan** [comment]
Thanks, appreciate the feedback! I've updated the README.md, you're correct, I put in the wrong folder.

I also pushed change for what I think will make Grey's Anatomy search working. I have to head to work soon, I haven't had a chance to test it out, can you try it out and report back?

I was also thinking of moving the contents of addic7ize method into a separate file. Perhaps Addic7edUtilities.py file.. haven't decided but something will be made ideal before too long. ;)

**Comment by Amelandbor** [comment]
Thanks for the quick update, but unfortunately it isn't working. It gives no results back and in the debug log isn't much info.

It seems in the code an apostrophe is deleted while it has to remain as an html character in the url we're searching. See the end of line 66

```
name = addic7ize(name).lower().replace(" ", "_").replace("$#*!","shit").replace("'","") 
```

**Comment by cflannagan** [comment]
I see that now. I dug up history (to find the original reasoning for doing that) and it appears it was added way back in December 2010, just for "That '70's Show"

https://github.com/amet/script.xbmc.subtitles/commit/2124e9f4362bcb934a1dd8597e0da8572b4ff66a

I checked to see what file naming would be used for majority of shows under "That 70s Show", "Shit My Dad Says", and it appears the chained replace calls is outdated today based on what I'm currently seeing on Addic7ed website, so they were removed.

I tested the change out for Grey's Anatomy and it seems to be working now. Let me know if you run into any more issues for Grey's Anatomy searches. Closing for now.
